### DISCUSSION TITLE
Api changes

### DISCUSSION THREAD

**Discussion by jonwalz** [jonwalz/crypto-data#11] [General]
I was messing around with the sanitment api and realizing there are some limitations. We may need to consider a different api. I've been looking at https://nomics.com/docs/#operation/getExchangeMarketsTicker

Namely, I noticed we can't do a simple search with santiment. Currently it just fetching the first page of all currencies then we filter through that on the frontend. We need and API with witch we can search currencies with partial match such as "BT" rather than "BTC".

**Comment by jonfreihofer** [comment]
Yeah, I kind of got that vibe after playing with it some more too. The whole reason we went with santiment was to mess with graphQL some more if I'm not mistaken, but if we want more flexibility nomics seems appropriate

**Comment by jonwalz** [comment]
Just found this. Could be a contender https://www.coingecko.com/en/api/documentation

**Comment by jonwalz** [comment]
We should hash out which kinds of features we'll need from an API to make a decision

**Comment by jonwalz** [comment]
https://github.com/nomics-crypto/nomics-javascript
